eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Certs.hs

```diff
@@ -254,3 +254,28 @@ instance
   where
   wrapFailed = CertFailure
   wrapEvent = CertEvent
+
+-- | Update dormant expiry for all DReps that are active.
+-- And also reset the `numDormantEpochs` counter.
+updateDormantDRepExpiry ::
+  -- | Current Epoch
+  EpochNo ->
+  -- | Number of dormant epochs
+  EpochNo ->
+  VState era ->
+  VState era
+updateDormantDRepExpiry currentEpoch numDormantEpochs vState =
+  if numDormantEpochs == EpochNo 0
```

Comment:
We are already checking this condition before calling the function, no?  Is there a point to check both before and within the function?

Reply:
If we're checking inside the function, then maybe we don't need to pass the `numDormantEpochs` as a parameter, since it's already inside the VState?